app/composer.json

```diff
@@ -119,14 +119,6 @@
     "psr/simple-cache": ">=3.0.0"
   },
   "repositories": [
-    {
-      "type": "git",
-      "url": "https://github.com/mautic/FOSOAuthServerBundle.git"
-    },
-    {
-      "type": "git",
-      "url": "https://github.com/mautic/SpBundle.git"
-    },
```

Comment:
This is amazing that we can ditch 2 forks we are managing! Both of the packages also support Symfony 6 and 7 so that's going to help us with maintenance a lot! 👍